Sources/swift-build/main.swift

```diff
@@ -67,7 +67,7 @@ do {
         usage()
 
     case .Clean(.Dist):
-        try rmtree(opts.path.Packages)
+        if opts.path.Packages.isDirectory { try rmtree(opts.path.Packages) }
```

Comment:
I think it would be good to have comments here (and below) about why the `if` is useful.